web/router.ex

```diff
@@ -33,19 +34,24 @@ defmodule Entice.Web.Router do
   scope "/api", Entice.Web do
     pipe_through :api
 
-    post "/login",            AuthController, :login
-    post "/logout",           AuthController, :logout
+    post    "/login",                 AuthController,     :login
+    post    "/logout",                AuthController,     :logout
 
-    get  "/char",             CharController, :list
-    post "/char",             CharController, :create
+    get     "/char",                  CharController,     :list
+    post    "/char",                  CharController,     :create
 
-    get  "/maps",             DocuController, :maps
-    get  "/skills",           DocuController, :skills
-    get  "/skills/:id",       DocuController, :skills
+    get     "/maps",                  DocuController,     :maps
+    get     "/skills",                DocuController,     :skills
+    get     "/skills/:id",            DocuController,     :skills
 
-    get  "/token/entity",     TokenController, :entity_token
+    get     "/token/entity",          TokenController,    :entity_token
 
-    post "/account/register", AccountController, :register
-    post "/account/request",  AccountController, :request_invite
+    get     "/account/by_char_name",  AccountController,  :by_char_name
+    post    "/account/register",      AccountController,  :register
+    post    "/account/request",       AccountController,  :request_invite
+
+    get     "/friend",                FriendsController,   :index
```

Comment:
Minor thingy: last part is not aligned with the ones above... if we want to be so picky ^^